v0/destinations/pardot/data/PardotIdentify.json

```diff
@@ -0,0 +1,277 @@
+[
+    {
+        "destKey": "email",
+        "sourceKeys": [
+      "traits.email",
+      "context.traits.email"
+        ],
+        "required": true
+      },
+      {
+        "destKey": "campaign_id",
+        "sourceKeys": [
```

Comment:
Ideally `campaign_id` is a part of the UTM params. @sayan-mitra Can you confirm?

Reply:
what should we do with this then?